### DISCUSSION TITLE
正體中文(台灣)v1 完整版

### DISCUSSION THREAD

**Discussion by austinwen98** [aclap-dev/video-downloadhelper#1572] [Language & Translation]
[zh_TW_v1.txt](https://github.com/user-attachments/files/18258008/zh_TW_v1.txt)

**Comment by slchoong** [comment]
[zh_TW.txt](https://github.com/user-attachments/files/19396646/zh_TW.txt)

**Comment by Tadashi1024** [comment]
[zh_TW.txt](https://github.com/user-attachments/files/20265883/zh_TW.txt)

Version 9.5.0.1
